<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userpanel</class>
 <widget class="QWidget" name="userpanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>441</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Панель пользователя</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnIndTrU">
      <property name="text">
       <string>Индивидуальные тренировки</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnTrainerU">
      <property name="text">
       <string>Тренеры</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSectionU">
      <property name="text">
       <string>Секции</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnVisitingU">
      <property name="text">
       <string>Посещение</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnStudentU">
      <property name="text">
       <string>Ученики</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPlaceU">
      <property name="text">
       <string>Место проведения</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Сменить пользователя</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
